### ISSUE TITLE
Opengl Polygram Corners Issue

### ISSUE THREAD

**Issue by ryanmccauley211** [ManimCommunity/manim#2860]
## Description of bug / unexpected behavior

In opengl the polygram example shows some corners are not rendered correctly when the width is increased. This may point to an issue in the shaders.

## Expected behavior

Six even corners in the hexagon

## How to reproduce the issue

Set the renderer to opengl 

<details><summary>Code for reproducing the problem</summary>

```py
class PolygramExample(Scene):
    def construct(self):
        hexagram = Polygram(
            [[0, 2, 0], [-np.sqrt(3), -1, 0], [np.sqrt(3), -1, 0]],
            [[-np.sqrt(3), 1, 0], [0, -2, 0], [np.sqrt(3), 1, 0]], stroke_width=20.
        )
        self.add(hexagram)
```

It only seems to cause an issue with two groups of vertices, rendering only one group results in a triangle with all corners connected correctly.

</details>

## Additional media files

<details><summary>Images/GIFs</summary>

**Cairo Render**

![PolygramExample_ManimCE_v0 15 2](https://user-images.githubusercontent.com/32387857/177408138-289a343c-5768-4e2e-b8c5-498ee8abefda.png)

**Opengl Render**

![PolygramExample_ManimCE_v0 15 2](https://user-images.githubusercontent.com/32387857/177408243-b22240cd-42b2-461b-bfab-a5043ad407ea.png)

**Opengl Render Wireframe (with blending disabled)**

![PolygramExample_ManimCE_v0 15 2](https://user-images.githubusercontent.com/32387857/177408399-523f1ce7-0a84-40f1-848d-e137bfddc82e.png)

</details>

## Logs
<details><summary>Terminal output</summary>

```
PASTE HERE OR PROVIDE LINK TO https://pastebin.com/ OR SIMILAR
```

</details>

## System specifications

<details><summary>System Details</summary>

- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)):
- RAM:
- Python version (`python/py/python3 --version`):
- Installed modules (provide output from `pip list`):
```
PASTE HERE
```
</details>

<details><summary>LaTeX details</summary>

+ LaTeX distribution (e.g. TeX Live 2020):
+ Installed LaTeX packages:

</details>

<details><summary>FFMPEG</summary>

Output of `ffmpeg -version`:

```
PASTE HERE
```
</details>

**Comment by MrDiver** [comment]
Yep because we do not have a polyline shader. So we are not able to choose between the different cap modes that are normally supported in cairo. Instead we just get seperate bezier curves forming a shape.
